v2/internal/controllers/documentdb_mongodb_crud_v20231115_test.go

```diff
@@ -0,0 +1,240 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	documentdb "github.com/Azure/azure-service-operator/v2/api/documentdb/v1api20231115"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+)
+
+func Test_DocumentDB_MongoDatabase_v20231115_CRUD(t *testing.T) {
+	t.Parallel()
+	tc := globalTestContext.ForTest(t)
+
+	// Create our resource group
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	// Declare a Cosmos DB account for our MongoDB database
+	kind := documentdb.DatabaseAccount_Kind_Spec_MongoDB
+	offerType := documentdb.DatabaseAccountOfferType_Standard
+	acct := documentdb.DatabaseAccount{
+		ObjectMeta: tc.MakeObjectMetaWithName(tc.NoSpaceNamer.GenerateName("db")),
+		Spec: documentdb.DatabaseAccount_Spec{
+			Location: to.Ptr("australiaeast"), // Capacity constraints //  tc.AzureRegion,
```

Comment:
minor, you can actually edit `tc.AzureRegion = to.Ptr("australiaeast")` and then have the code just look "standard" but with the edit. This had the advantage of putting all the other resources in the same region too.

If you do this now though you'd have to either manually edit the recordings (which actually would be pretty easy with find-replace) or re-record as the RG and whatever other resources would be in a new spot.